packages/rich-text-editor/src/RichTextEditor/components/Toolbar/Toolbar.scss

```diff
@@ -0,0 +1,7 @@
+@import "@kaizen/design-tokens/sass/color";
+@import "@kaizen/design-tokens/sass/spacing";
+
+.toolbar {
+  display: block;
+  border-bottom: 2px solid $color-gray-600;
```

Comment:
This border could belong here or in the actual text input wrapper as a border top. Either way one consideration is how we handle the colour on the error styles - I've left it out for now till we have all the components